modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -245,6 +245,7 @@
             <class name="org.wso2.identity.integration.test.user.profile.mgt.UserProfileAdminTestCase"/>
             <!-- TODO this test has a restart-->
             <class name="org.wso2.identity.integration.test.auth.RiskBasedLoginTestCase" />
+            <class name="org.wso2.identity.integration.test.identity.mgt.AccountLockWhileCaseInsensitiveUserFalseTestCase"/>
```

Comment:
As this test case contains a re-start this test should be fall into the 'is-tests-with-individual-configuration-changes' group.